ec2uploadimg

```diff
@@ -333,8 +334,8 @@ if not args.useSnap:
 try:
     utils.check_account_keys(config, args)
 except Exception:
-    print_ex()
-    sys.exit(1)
+    logger.debug('Could no find account keys in "%s", \
+                 will check aws config files' % config_file)
```

Comment:
Given the new structure and what the other utils routines do I think we no longer need `utils.check_account_keys()` but that can be handled in a separate request.

Reply:
Robert,  for this separate request do we need to unit test ec2utils.conf entries as well as the aws entries or just modify the existing ec2utils.conf tests to the new structure?

Reply:
OK,....

-> find . -type f | xargs grep check_account_keys | grep -v test
./lib/ec2imgutils/ec2utils.py:def check_account_keys(config, command_args):
./ec2uploadimg:    utils.check_account_keys(config, args)

shows that `check_account_keys()` is used in only 1 place, here. Then if I look at the implementation of `check_account_keys()` in `ec2utils.py` the doc string tells me `Verify that API access keys are available` and the implementation is pretty clear that it only looks for `access_key_id` and `secret_access_key`. The intention of the implementation of `check_account_keys()` is pretty clear in that it tries to assert that we never get in a situation where only one or none of the keys are available.

But that condition is already covered with the way the implementation below:

```
if not access_key:
    logger.error('Could not determine account access key')
    sys.exit(1)

....

if not secret_key:
    logger.error('Could not determine account secret access key')
    sys.exit(1)
```

So the way things are implemented now `check_account_keys()` is superfluous and can be removed from here, the ec2utils implementation and the unit tests. Now of course that I did all the investigation the rest is 5 extra minutes that I also could have invested.

Reply:
well it might have all been avoided, if it was clear in the review of my first iteration i could drop it from ec2utils and unit tests. I got the impression those needed to stay. away they go